### DISCUSSION TITLE
Configuración de Zonas de envio

### DISCUSSION THREAD

**Discussion by Gerdetect** [whooohq/whq-woocommerce-chilexpress-shipping#243] [Support]
Hola, les escribo porque necesito de su ayuda, instale el plugin, me aparecen todas las regiones y comunas. Pero no me esta haciendo el calculo de los envíos. Creo que esto esta pasando porque en la configuración de las zonas de envió no me da la opción de configurar como método de envió (chilexpress) y por ello no me estaría haciendo el calculo. Si pudieran ayudarme por favor. Gracias!!!
![Captura de pantalla 2022-05-10 145348](https://user-images.githubusercontent.com/105314673/167701779-6f32e6a9-eb42-42f9-89b6-d452ac69ee10.png)

**Comment by EstebanForge** [comment]
Estimado, buen día.

El plugin NO es compatible con las zonas de envío. Ver: https://github.com/whooohq/whq-woocommerce-chilexpress-shipping/issues/39

Aclarado aquello, el plugin está funcionando.
Lo puedes ver, andando correctamente, en https://wpchilexpress.whooohq.com/
Pantallazo: https://imgur.com/KxzO46c

Para que el plugin funcione, eso si, debes considerar esto: https://github.com/whooohq/whq-woocommerce-chilexpress-shipping#c%C3%A1lculo-de-precio-de-env%C3%ADo

Quedo atento.

Esteban

**Reply by Gerdetect** [reply]
Hola gracias por tu pronta respuesta, también hice la prueba sin ninguna zona de envió, y sigue sin hacer el calculo. Todos los productos tienen configuradas sus medidas y peso.
